relay-filter/src/config.rs

```diff
@@ -141,12 +141,14 @@ pub struct ErrorMessagesFilterConfig {
 pub struct IgnoreTransactionsFilterConfig {
     /// List of patterns for ignored transactions that should be filtered.
     pub patterns: GlobPatterns,
+    /// True if the filter is enabled
+    pub is_enabled: bool,
```

Comment:
There is a tiny chance that somebody updated their external Relay since this filter was introduced. These old Relays would ignore this flag until they update again. But given the short time frame and the fact that there was no calendar release in between, I think we can ignore this problem.